### ISSUE TITLE
Two similar custom environment, PPO learns on both but SAC only on one

### ISSUE THREAD

**Issue by tfederico** [hill-a/stable-baselines#1191]
**Describe the bug**
I have two custom environments, one with a hand and one with a humanoid with hands. In the past, I trained the humanoid with just PPO and the hand with both PPO and SAC. 
I am currently trying to train the humanoid with SAC by adapting the training script I used for the hand. However, it does not converge at all. I am trying several hyperparams combinations, but I always get the same results no matter the combination. The maximum reward per step is 1 and the cumulative reward I am getting after 2000 steps is always 1. However, if I use the *exact same* environment but train using PPO instead of SAC, it works. Also, if I use the exact same training script with the hand environment, it works.

Any idea about what could be wrong?

**Code example**
[Repository with code](https://github.com/tfederico/MimicTest)

**System Info**
Describe the characteristic of your environment:
 * GPU model: RTX 2080Ti
 * Python version 3.9.12
 * Versions of any other relevant libraries: all indicated in requirements.txt file

**Comment by tfederico** [comment]
To add more info, I am currently running a hyperparameters search over 5M steps on the following parameters

```
parameters:
  learning_rate:
    values: [0.003, 0.001, 0.0003, 0.0001, 0.00003]
  buffer_size:
    values: [1000000, 2000000]
  learning_starts:
    values: [100, 5000, 10000]
  batch_size:
    values: [256, 512]
  tau:
    values: [0.0, 0.1, 0.25, 0.5]
  ent_coef:
    values: ["auto", "auto_0.1", "0.25", "0.5", "1.0"]
  target_entropy:
    values: ["auto", 0.1, 0.2]
  log_std_init:
    values: [-1.0, -3.0, -5.0]
```
